### DISCUSSION TITLE
Was banned made unban request and not a real person would read it

### DISCUSSION THREAD

**Discussion by ilovegaming1008** [citizenfx/fivem#1857] [Engineering Discussion]
I was banned about a year ago by now for I belive 265 days or something or 180 I forgot off a hardware ban when I didnt even do anything and a lot of other people it happened to as well we all waited so long to play it again but than I got rebanned right after the timer so please do something about this my discord is ilove#7647 I can show proof of everything because I screanshotted and I have chats with friends when it happened thanks.

**Comment by thorium-cfx** [comment]
Please refrain yourself from posting unban requests on technical support channels, thank you.
